acslib/ccure/types.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict
+
+
+class CredentialCreateData(BaseModel):
```

Comment:
I can add a docstring that says what you wrote above, but I don't know anything else about credentials in ccure.

Reply:
Do you have access to the Test CCURE instance? Since we are developing a library for general use, not just internal, we should be the subject matter experts on what we are providing. We don't need to have comprehensive knowledge, but if we are pushing out a feature we should know how it functions within the overall system, and be able to describe its impact on when used in the way we are implementing.  Some questions that come to mind are: Why are these fields and these fields only the ones that you chose for this feature implementation?

Reply:
I took a stab at this, let me know what you think. More detail?

Reply:
My detail brain wants more, but honestly it sounds like this gets at the essence of what is required.